tests/test3/task2/list.c

```diff
@@ -0,0 +1,143 @@
+#define _CRT_SECURE_NO_WARNINGS
+
+#include <stdio.h>
+#include <string.h>
+
+#include "list.h"
+
+#define MAX_SIZE 256
+
+typedef struct Node {
+    char* value;
+    struct Node* next;
+} Node;
+
+typedef struct List {
+    int length;
+    Node* head;
+    Node* tail;
+} List;
+
+List* createList(void) {
+    List* list = malloc(sizeof(List));
+    if (list == NULL) {
+        return NULL;
+    }
+    list->head = NULL;
+    list->tail = NULL;
+    list->length = 0;
+    return list;
```

Comment:
```suggestion
    return calloc(1, sizeof(List));
```
Это буквально то же самое, но короче